modules/cnf-about-collecting-nro-data.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-collecting-nro-data_{context}"]
+= About collecting NUMA-aware pod scheduling data
```

Comment:
The title is throwing me off a bit. Seems like "About" shouldn't be at the end of a section. This is also formatted as a procedure, so how about just starting with the verb?
```suggestion
= Collecting NUMA-aware pod scheduling data
```